scripts/migration/migrate_registered_meta.py

```diff
@@ -0,0 +1,80 @@
+""" Changes existing registered_meta on a node to new schema layout
+required for the prereg-prize
+"""
+import re
+import json
+import sys
+import logging
+
+from modularodm import Q
+
+from website import models
+from website.app import init_app
+from scripts import utils as scripts_utils
+
+logger = logging.getLogger(__name__)
+logger.setLevel(logging.INFO)
+IMMUTABLE_KEYS = frozenset(['datacompletion', 'summary'])
+
+def get_registered_nodes():
+    return models.Node.find(
+        Q('is_registration', 'eq', True)
```

Comment:
We may need to run the script multiple times, e.g. if the script crashes or we need to stop the script pre-maturely. Ideally, migration scripts should be idempotent (running multiple times with the same database will produce the same result) and resumable.

Do we need to migrate every registration every time we run the script?